src/main/java/Duke/Deadline.java

```diff
@@ -0,0 +1,36 @@
+package Duke;
+import java.text.SimpleDateFormat;
+
+ /**
+  * <pre>
+  * Child class Deadline from Task Class
+  * </pre>
+  */
+public class Deadline extends Task{
+
+
+    /**
+     * Defult constructor,
+     */
+    public Deadline(String actionName, boolean compleated, String date, String type) {
```

Comment:
I think you might have typo-ed completed as compleated here. Also, the coding standard suggests using prefixes like is, has, and was for boolean variable/method names - do you want to change this to `isCompleted`?